src/components/Button/Button.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import styles from './Button.module.scss';
+import { Loader, LoaderSize } from '../Loader/Loader';
+
+export type ButtonProps = React.PropsWithChildren<{
+  loading?: boolean;
+  children?: React.ReactNode;
+  classname?: string;
```

Comment:
camelCase пожалуста..
```suggestion
  className?: string;
```